### ISSUE TITLE
Feature request: view attachments

### ISSUE THREAD

**Issue by bhankerson** [twsouthwick/KeePassWin#60]
Is what would set this app apart from other KeePass managers

**Comment by twsouthwick** [comment]
This would be nice. Are attachments text files, other files, or links generally? Or can they be anything?

**Comment by bhankerson** [comment]
Hi Taylor,

In KeePass 2.x, I can attach files to entries. I use it specifically to attach images of my cards to the entry (jpg, bmp, png). I'm not sure how many other types are supported (pdf isn't, I know). This would be extremely useful in situations like I had a couple months ago where my wallet was stolen and I needed information that was on the cards (stupid system for stolen credit cards, but there you have it).

Thanks,

Brett

________________________________

**Comment by GitMoDu** [comment]
Yes, a nice preview for images and simple export file are features needed, if KeePassWin wants to be able to fully replace KeePass old-school.
